src/Maestro/maestro-angular/src/app/page/build-graph-table/build-graph-table.component.html

```diff
@@ -109,11 +117,17 @@
         <fa-icon [icon]="!isCoherent(node) ? 'exclamation-circle' : 'exclamation-triangle'" 
           [title]="!isCoherent(node) ? 'A newer build of this same repository is in the tree' : (hasIncoherentDependencies(node) ? 'This build relies on dependencies that are incoherent' : '')" 
           [style.visibility]="(!isCoherent(node) || hasIncoherentDependencies(node)) ? 'visible' : 'hidden'"></fa-icon>
+        <fa-icon [icon]="hasCycle(node) ? 'redo' : 'exclamation-triangle'" 
+          [title]="hasCycle(node) ? 'Build contains a dependency cycle: \n' + cyclePath(node) : ''" 
+          [style.visibility]="hasCycle(node) ? 'visible' : 'hidden'"></fa-icon>
         {{node.build.azureDevOpsBuildNumber}}
       </td>
       <td class="text-monospace">
         <fa-icon [icon]="!isCoherent(node) ? 'exclamation-circle' : 'exclamation-triangle'" 
         [style.visibility]="(!isCoherent(node) || hasIncoherentDependencies(node)) ? 'visible' : 'hidden'"></fa-icon>
+        <fa-icon [icon]="hasCycle(node) ? 'redo' : 'exclamation-triangle'" 
```

Comment:
I don't know that two icons (in different columns) are needed to display a cycle

Reply:
I see what you're saying. I put it in both, because we put the coherency symbol in both places. I can remove one of them if that would be clearer.